verification/curator-service/ui/src/components/Alerts.test.tsx

```diff
@@ -0,0 +1,81 @@
+import Alerts from './Alerts';
+import React from 'react';
+import axios from 'axios';
+import { render } from '@testing-library/react';
+
+jest.mock('axios');
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+afterEach(() => {
+    jest.clearAllMocks();
+});
+
+it('loads and displays alerts', async () => {
+    const uploads = {
+        uploads: [
+            {
+                _id: '5ef8e943dfe6e00030892d58',
+                status: 'IN_PROGRESS',
```

Comment:
Do we want to filter out IN_PROGRESS? Displaying only SUCCESS/ERROR.

Reply:
Yes, we should also filter for actioned uploads. Currently thinking that's best to do in the API endpoint, so will leave for a future PR.